language/stdlib/modules/vector.mvir

```diff
@@ -5,9 +5,17 @@ module Vector {
   // Vector containing data of type Item
   native struct T<Element>;
 
+  native public empty<Element>(): Self.T<Element>;
+
   // Return the length of the vector
   native public length<Element>(v: &Self.T<Element>): u64;
 
+  // Acquire an immutable reference to the ith element of the vector
+  native public borrow<Element>(v: &Self.T<Element>, i: u64): &Element;
+
+  // Add an element to the end of the vector
+  native public push_back<Element>(v: &mut Self.T<Element>, e: Element);
```

Comment:
Do we want to just use `push` like in Rust?

Reply:
I always hated Rust's `push` name because it sounds like it might be adding an element to the front. `push_back` makes it really explicit where the element is going, which I think is a good thing.

Reply:
I tend to agree. Even though I have used `push` a dozen times, I'm always checking the Rust docs page to see if it goes to the front or the back :p